services/appflowy-collaborate/src/collab/storage.rs

```diff
@@ -75,6 +89,103 @@ where
     }
   }
 
+  const PENDING_WRITE_BUF_CAPACITY: usize = 100;
```

Comment:
Maybe we can use env here. It's easy to adjust the capacity

Reply:
I didn't put it into configurable value since it's also hardcoded in the current implementation: see `consume_pending_write(&pending_write_set, 20, 5).await;` in the deleted `queue.rs` file.

Reply:
I think you might have misunderstood what the source code does—it writes 20 collabs at a time.